tests/good_output/test.sh

```diff
@@ -1,16 +1,10 @@
 #!/bin/sh
 
-###########################################
-#             Test - distar               #
-#           Copyright (C) 2018            #
-#   Yann Régis-Gianas - Étienne Marais    #
-###########################################
-
-
-##  Test if the simple output is good
+set -euC
+cd "$(dirname "$0")"
 
 # Program output
```

Comment:
Are you sure that `distar` exists at this point? Imagine I have just cloned the repository and run `make check`, is `distar` built before the testsuite is run?

Reply:
Yes it is build by the jbuilder command : 
```jbuilder
(deps ((file ../src/distar.exe)
```

Reply:
Great!